de.dashup.shared/src/main/java/de/dashup/shared/DatabaseModels/DatabaseSectionWidgets.java

```diff
@@ -0,0 +1,44 @@
+package de.dashup.shared.DatabaseModels;
+
+import com.google.gson.annotations.SerializedName;
+
+public class DatabaseSectionWidgets extends DatabaseObject implements Order{
+
+    private Integer id;
+    @SerializedName("section_id")
+    private Integer sectionID;
+    @SerializedName("widget_id")
+    private Integer widgetID;
+    @SerializedName("predecessor_id")
+    private Integer predecessorID;
```

Comment:
We decided to remove the `predecessor` and introduce the `order` again. This was also a task of @svenleonhard, which will cause merge conflicts.

Reply:
As mentioned above, there was a lack of communication, unfortunately.